slink-main/config.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const awsParamStore = require('aws-param-store');
+
+const configParams = {
```

Comment:
Maybe just call it `params` since it's already in a module called `config`?  Makes code shorter.